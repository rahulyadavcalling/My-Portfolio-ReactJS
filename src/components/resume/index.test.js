import React from "react";
import renderer from "react-test-renderer";

import Resume from "./index.js";

it("renders correctly when there are no items", () => {
  const tree = renderer.create(<Resume />).toJSON();
  expect(tree).toMatchSnapshot();
});
