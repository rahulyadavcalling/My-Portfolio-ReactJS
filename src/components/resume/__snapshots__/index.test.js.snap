// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no items 1`] = `
<div
  className="sc-bdVaJa gYXdsf"
>
  <img
    alt="react-boilerplate - Logo"
    className="animated zoomIn delay-0.2s sc-htpNat iLHwDe"
    src="Reyansh.png"
  />
  <a
    className="animated slideInUp delay-0.3s sc-bwzfXH llauBH"
    href="reyansh_mishra.pdf"
    target="_blank"
  >
    Download
  </a>
</div>
`;
